### PR TITLE
Add Mojave themes initial commit

### PR DESCRIPTION
**Pull request by fredcw**
Add Mojave-Dark and Mojave-Light themes.

### CODE REVIEW

File: Mojave-Light/info.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+	"author": "fredcw",
```
Comment (rcalixte) [inline Mojave-Light/info.json:2]: It may make sense to start adding an `original_author` item here with attribution to the source authors?

**Review by rcalixte — APPROVED** [review]

File: Mojave-Light/info.json (hunk 1 above)
Comment (fredcw) [reply Mojave-Light/info.json:2]: Yes, I did consider this. The author has given me consent to upload his themes to cinnamon but he didn't specifically state that he wanted his github name to be associated with the upload so I thought it best not to associate his name with it unless he wants that. Also, I thought that maybe the 'original_author' field may be interpreted as the person who originally uploaded the theme or was the original maintainer of the cinnamon spice. Of course, the author is welcome to put his name as either the `author` or `original author` any time he wishes.

Thanks for the quick merges btw, it really does make all the difference in terms of being motivated to contribute to cinnamon.

Comment (rcalixte) [reply Mojave-Light/info.json:2]: > Yes, I did consider this. The author has given me consent to upload his themes to cinnamon but he didn't specifically state that he wanted his github name to be associated with the upload so I thought it best not to associate his name with it unless he wants that. Also, I thought that maybe the 'original_author' field may be interpreted as the person who originally uploaded the theme or was the original maintainer of the cinnamon spice. Of course, the author is welcome to put his name as either the `author` or `original author` any time he wishes.

Will you be monitoring the sources for changes and submitting those as well?



> Thanks for the quick merges btw, it really does make all the difference in terms of being motivated to contribute to cinnamon.

This is an awesome community and it's great to be able to contribute.

Comment (fredcw) [reply Mojave-Light/info.json:2]: Yeah, that's the plan, to keep them updated on a reasonably regular basis. Actually, that reminds me, I should put the theme version number or release date in the readme.

Comment (rcalixte) [reply Mojave-Light/info.json:2]: > Yeah, that's the plan, to keep them updated on a reasonably regular basis.

That's awesome and thank you. I'm already playing around with some of these.



> Actually, that reminds me, I should put the theme version number or release date in the readme.

Assuming you're grabbing the latest versions, hopefully the git commit log is useful for auto-tracking that. If not, feel free to add something in the README and/or info.json files.

**Pull request merged**